<ui version="4.0" >
 <author>Brian S. Stephan</author>
 <class>SweeperDialog</class>
 <widget class="QWidget" name="SweeperDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>480</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QSplitter" name="splitter1" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Expanding" hsizetype="Expanding" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QTreeWidget" name="privacyListView" >
      <property name="whatsThis" >
       <string>Check all cleanup actions you would like to perform. These will be executed by pressing the button below.</string>
      </property>
      <property name="alternatingRowColors" >
       <bool>true</bool>
      </property>
      <property name="itemsMovable" stdset="0" >
       <bool>false</bool>
      </property>
      <column>
       <property name="text" >
        <string>Privacy Settings</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>Description</string>
       </property>
      </column>
     </widget>
     <widget class="KTextEdit" name="statusTextEdit" >
      <property name="readOnly" >
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <widget class="QPushButton" name="selectAllButton" >
       <property name="text" >
        <string>Select &amp;All</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="selectNoneButton" >
       <property name="text" >
        <string>Select &amp;None</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType" >
        <enum>QSizePolicy::MinimumExpanding</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>171</width>
         <height>16</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="cleanupButton" >
       <property name="whatsThis" >
        <string>Immediately performs the cleanup actions selected above</string>
       </property>
       <property name="text" >
        <string>&amp;Clean Up</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
  <customwidgets>
  <customwidget>
   <class>KTextEdit</class>
   <extends>QTextEdit</extends>
   <header>ktextedit.h</header>
  </customwidget>
 </customwidgets>
 <includes>
  <include location="local" >ktextedit.h</include>
 </includes>
 <resources/>
 <connections/>
</ui>
